tester/server/server.cpp

```diff
@@ -120,23 +117,12 @@ bool	server::fd_ready(void)
 	return 1;
 }
 
-void	server::add_user(int fd, sock_in client_addr)
```

Comment:
Yo eso no lo he tocado uwu igual al estar en la carpeta de testers no está totalmente actualizado

Reply:
Intenta hacer un rebase
 Git rebase --master

Reply:
Eso no me va a borrar mis cosas no? uwu

Reply:
https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase

Reply:
`The primary reason for rebasing is to maintain a linear project history. For example, consider a situation where the main branch has progressed since you started working on a feature branch. You want to get the latest updates to the main branch in your feature branch, but you want to keep your branch's history clean so it appears as if you've been working off the latest main branch`

Reply:
He hecho un cambio así to guarro uwu